cob_twist_controller/include/cob_twist_controller/cob_twist_controller.h

```diff
@@ -137,24 +111,20 @@ class CobTwistController
         this->limiters_.reset();
     }
 
+    bool initialize();
     void run();
 
-    bool initialize();
     void reinitServiceRegistration();
 
     void reconfigureCallback(cob_twist_controller::TwistControllerConfig &config, uint32_t level);
     void jointstateCallback(const sensor_msgs::JointState::ConstPtr& msg);
     void odometryCallback(const nav_msgs::Odometry::ConstPtr& msg);
+    
     void twistCallback(const geometry_msgs::Twist::ConstPtr& msg);
-    void baseTwistCallback(const geometry_msgs::Twist::ConstPtr& msg);
     void twistStampedCallback(const geometry_msgs::TwistStamped::ConstPtr& msg);
 
     void solveTwist(KDL::Twist twist);
 
-    ///Debug
-    void showMarker(int marker_id,double red, double green, double blue, std::string ns, ros::Publisher pub, std::vector<geometry_msgs::Point> &pos_v);
-    void debug();
```

Comment:
do we still want some of the debug stuff back in?

split into DEBUG_BASE_COMPENSATION (goes into twist_controller.cpp) and DEBUG_BASE_ACTIVE (goes into kinematic_extension.cpp/KinematicExtensionBaseActive class)